build.gradle

```diff
@@ -197,25 +198,138 @@ tasks.build.dependsOn makeMdk
 
 // launch4j
 
-launch4j {
-    jar = installer.archivePath.canonicalPath
-    outfile = file("build/distributions/${project.name}-${project.version}-installer-win.exe").canonicalPath
-    icon = file('icon.ico').canonicalPath
-    manifest = file('l4jManifest.xml').canonicalPath
-    jreMinVersion = '1.6.0'
-    initialHeapPercent = 5;
-    maxHeapPercent = 100;
+//launch4j {
+//    jar = installer.archivePath.canonicalPath
+//    outfile = file("build/distributions/${project.name}-${project.version}-installer-win.exe").canonicalPath
+//    icon = file('icon.ico').canonicalPath
+//    manifest = file('l4jManifest.xml').canonicalPath
+//    jreMinVersion = '1.6.0'
+//    initialHeapPercent = 5;
+//    maxHeapPercent = 100;
+//}
+//tasks.generateXmlConfig.dependsOn installer
+//tasks.build.dependsOn 'launch4j'
+
+//Wrapper tasks
+repositories {
+    maven {
+        name = "forge maven"
+        url = "http://files.minecraftforge.net/maven"
+    }
+}
+
+configurations {
+    wrapTar
+    wrapZip
+}
+
+dependencies {
+    wrapZip "net.minecraftforge.javawrapper:javawrapper:$wrapper_version:win@zip"
+    wrapTar "net.minecraftforge.javawrapper:javawrapper:$wrapper_version:linux@tar.gz"
+    wrapTar "net.minecraftforge.javawrapper:javawrapper:$wrapper_version:osx@tar.gz"
 }
-tasks.generateXmlConfig.dependsOn installer
-tasks.build.dependsOn 'launch4j'
+
+def wrapLinux = "javawrapper-v$wrapper_version-linux"
+def wrapOsx = "javawrapper-v$wrapper_version-osx"
+def wrapWin = "javawrapper-v$wrapper_version-win.exe"
+
+def wrapLinuxPath = file("$buildDir/jwrappers/$wrapLinux")
+def wrapOsxPath = file("$buildDir/jwrappers/$wrapOsx")
+def wrapWinPath = file("$buildDir/jwrappers/$wrapWin")
+
+task wrapInstaller(type: Copy) {
+    dependsOn tasks.installer, configurations.wrapTar, configurations.wrapZip
+
+    from zipTree(configurations.wrapZip.singleFile)
+    into "$buildDir/jwrappers"
```

Comment:
You should use the temporaryDir instead. That's what this type of thing is meant for.